web/queries/attendee_query.ex

```diff
@@ -1,10 +1,21 @@
 defmodule Syndeo.AttendeeQuery do
+  import Ecto.Query
   alias Syndeo.Repo
   alias Syndeo.Attendee
+  alias Syndeo.WeeklyInfo
 
   def find_attendee!(id) do
     Attendee
     |> Repo.get!(id)
     |> Repo.preload([:weekly_info])
   end
+
+  def find_attendees_with_current_week_info do
+    Repo.all(Attendee)
```

Comment:
For readability, most style guides recommend starting with a bare term, rather than a function:

```elixir
Attendee
|> Repo.all
|> Repo.preload(...)
```